lib/errors.js

```diff
@@ -23,7 +23,7 @@ function StatusCodeError(statusCode, body, options, response) {
 
     this.name = 'StatusCodeError';
     this.statusCode = statusCode;
-    this.message = statusCode + ' - ' + (JSON && JSON.stringify ? JSON.stringify(body) : body);
+    this.message = 'Request status code was not 2xx';
```

Comment:
Wouldn't it be better to at least mention the status code? Maybe you can add the default description (https://nodejs.org/api/http.html#http_http_status_codes)